nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/InvokeHTTP.java

```diff
@@ -367,6 +372,7 @@
                     + "on the normal truststore hostname verifier. Only valid with SSL (HTTPS) connections.")
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .required(false)
+            .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
```

Comment:
I'm not very comfortable with this field supporting EL either. Do you have a specific use case that requires it?